src/components/App.js

```diff
@@ -222,6 +222,7 @@ class App extends Component {
           onProgress={this.progressUpdate}
         />
         <div className="top-bar">
+          <i className="fa fa-2x fa-star" title="Favorite" />
```

Comment:
Should I make them divs instead?
Btw thanks so much for reviewing. :smile:

Reply:
In this instance, a span would probably be what you want. I'd usually lean towards simply adding the `fa` classes directly to the element that needs the icon, but its been a while since I've used fontawesome.

Reply:
So I'll convert them to span, but will it benefit keyboard users to tab over them?

Reply:
There would be no benefit if you were to use a span. But you could simply add the class to the button that needs an icon instead, which is better semantically. No need for an extra span if it does nothing but exist to hold an icon pseudoelement.

In regards to this specific `i` element, I'm not sure what the icon is even for. Looks like its just there for the sake of having an icon?

Reply:
> No need for an extra span if it does nothing but exist to hold an icon pseudoelement.

I think that was better(what you did) :sweat_smile: 

>In regards to this specific i element, I'm not sure what the icon is even for. Looks like its just there for the sake of having an icon?

If you checkout the current version https://ongaku.js.org/ this icons marks favourites for users, though that is yet to be done, I've just added it beforehand and will be completing it soon.

Reply:
OH I see! Yea, that should probably be a button too, but seeing as how the feature is not yet implemented, the icon as a placeholder will work fine too. The benefit is that a keyboard user can tab to the element and use <kbd>ENTER</kbd> or <kbd>SPACE</kbd> to trigger the click. There's more to accessibility than just that (for example, lets say a blind user enjoys this app, and wants to favorite a song. How can they know what the button will do? How would they know whether the song is already favorited or not?) I'd be happy to go into this more in depth when the feature is done though.

Reply:
Agreed, probably making it a placeholder will be the best possible choice. Plus if a song is already marked favourite it will show a filled star, if not, it will show a hollow star is what we've followed on the HTML repo. Also in preferences, there's an option added to only display the favourites.
I'll keep you updated with the progress.